.travis.yml

```diff
@@ -2,11 +2,7 @@ language: node_js
 os:
  - linux
 node_js:
-  - "11"
-  - "10"
-  - "8"
-  - "6"
-  - "4"
+  - "17"
```

Comment:
all of these must remain, otherwise it's semver-major. Instead, please revert and add 12-18.

Reply:
for node.js v11 and v10, the error msg:
```
ERROR: npm is known not to run on Node.js v10.24.1
You'll need to upgrade to a newer Node.js version in order to use this
version of npm. You can find the latest version at https://nodejs.org/
```

```
ERROR: npm is known not to run on Node.js v11.15.0
You'll need to upgrade to a newer Node.js version in order to use this
version of npm. You can find the latest version at https://nodejs.org/
```

is it ok to add those to allowed failure?

Reply:
for error in 4 and 18, not quite sure how to fix it, could you give some help?

Reply:
@duoertai, @ljharb  is it Ok to neglect the failed `node_js` versions by putting them in:
```
  allow_failures:
    - node_js: "iojs"
    - node_js: "0.12"
    - node_js: "0.10"
    - node_js: "11"
    - node_js: "10"
```

Reply:
I'd prefer not to; they should be passing. I'll take a look.